### PR TITLE
Tune sklearn

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge/help-python The CI fails on osx not because the package cannot be used on osx but because `ray` is currently not available for osx. Should I still add a skip for `osx` in that case? And just remove it from the feedstock once `ray` is available on `osx`?

**Comment by hadim** [comment]
@conda-forge/help-python please review

File: recipes/tune-sklearn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  # NOTE: pypi doesnt contain a source package for it. See
+  # https://github.com/ray-project/tune-sklearn/issues/159
+  url: https://github.com/ray-project/tune-sklearn/archive/v{{ version }}.tar.gz
+  sha256: 436cc5fa37d9cb596ce8a841b1c20120564d012114ebefd024aebdac753ce5f3
+
+build:
+  # noarch: python
+  number: 0
+  skip: true  # [osx]
```
Comment (dopplershift) [inline recipes/tune-sklearn/meta.yaml:17]: Better to remove this and make this `noarch:python`. The lack of `ray` on osx isn't a concern here. Also, if `ray` ever works on osx, this package will magically be installable.

Comment (hadim) [reply recipes/tune-sklearn/meta.yaml:17]: Yes, that's what I need first but the osx obviously fails. I guess it's fine?

Comment (hadim) [reply recipes/tune-sklearn/meta.yaml]: reverted back to noarch.

**Comment by hadim** [comment]
LGTM here. See https://github.com/conda-forge/staged-recipes/pull/13447#discussion_r541205970 for why osx is failing.

File: recipes/tune-sklearn/meta.yaml (hunk 1 above)
Comment (dopplershift) [reply recipes/tune-sklearn/meta.yaml]: Yeah, that's fine. Being noarch, it won't affect package builds in the feedstock (we use Linux to build noarch packages).  Mac users just don't get to install it.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**